<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Projects</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />
 <link rel="icon" type="image/png" href="static/favicon_io/favicon-32x32.png"/>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>






<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Telkar, N</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Welcome</a>
</li>
<li>
  <a href="projects.html">Projects</a>
</li>
<li>
  <a href="fun.html">Fun</a>
</li>
<li>
  <a href="cv.html">Academic CV</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://www.linkedin.com/in/nikitatelkar/">
    <span class="fa fa-linkedin"></span>
     
  </a>
</li>
<li>
  <a href="https://github.com/nikita-telkar">
    <span class="fa fa-github"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/nikita_telkar">
    <span class="fa fa-twitter"></span>
     
  </a>
</li>
<li>
  <a href="https://scholar.google.com/citations?hl=en&amp;authuser=2&amp;user=3EwE6tAAAAAJ/">Google Scholar</a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Projects</h1>

</div>


<style>
#TOC {
  background: url("static/TOC.png");
  background-size: contain;
  padding-top: 80px !important;
  padding-bottom: 80px !important;
  background-repeat: no-repeat;
}
</style>
<style>
a:link {
    color: purple;
}

a:visited {
    color: purple;
}

 a:hover {
    color: purple;
}

</style>
<style type="text/css">

@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;1,400&display=swap');
@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');

body {
    font-family: 'Open Sans';
    font-size: 15px;
}


h4, #TOC>ul>li {
  font-family: 'Montserrat';
}

# h5, #TOC>ul>ul>li {
#  color: #323b58;
# }

.tocify {
  # position: -webkit-sticky;
  # position: sticky; 
  # top: 120px; /*Controls where TOC stops when sticky */
  width: 100% !important;
  border: none;
}

.list-group-item {
    color: #666666;
    font-size: 15px;
}

.list-group-item.active {
    color: white;
    background-color: #853078;
    border: none;
}


.list-group-item:hover, 
.list-group-item.active:hover {
    color: #853078;
    background-color: white;
}

/*----------------NAVBAR ---------------*/
.navbar-default {
    background-color: #333333f2;
}

.navbar-default .navbar-nav>.open>a, 
.navbar-default .navbar-nav>.active>a, 
a.dropdown-toggle:hover {
  background-color: #333333 !important;
}

/* Dropdown menu color */
.navbar-default .dropdown-menu {
  background-color: #333333;
}

/* Dropdown menu hover color */
  .navbar-default .dropdown-menu>li>a:hover {
    background-color: #333333f2;
  }

/* Navbar Links when hovered*/
.navbar-default .dropdown-menu>.active>a
.navbar-default .navbar-nav>.active>a:hover, 
.navbar-default .navbar-nav:hover, 
.navbar-default .navbar-nav>li>a:hover, 
a.navbar-brand:hover {
  color: #ffffffab !important;
  background-color: #333333;
}
</style>
<hr />
<div id="phd-projects" class="section level3">
<h3>PhD Projects</h3>
<p><br></p>
<div id="primary-project" class="section level4">
<h4>Primary Project</h4>
<p><br></p>
<p><strong>Characterizing small non-coding RNAs in the human placenta</strong></p>
<p>The human placenta is the indispensible organ necessary for all water, nutrient, gas, and waste exchange between the mother and fetus; as such gene expression is tightly regulated. Gene regulation is carried out a number of mechanisms, and gene repression by small non-coding RNAs (sncRNAs) is one of these processes. <a href="https://10.1016/j.ajog.2015.07.046">Exclusive placentally-expressed sncRNA loci in the placenta have been identified</a>, however, studies have routinely focused on only one species of sncRNA (microRNAs).</p>
<p><strong>My project aims at</strong>:</p>
<ol style="list-style-type: decimal">
<li>Characterizing the sncRNA species that are present in the human placenta and 4 placental cell types (trophoblasts, endothelial cells, stromal cells, HofBauer cells)<br />
</li>
<li>Investigating novel sncRNA species<br />
</li>
<li>Cataloguing the ones that differ in expression by different variables (like trimester of the pregnancy, sex of the fetus, ancestry, depression and anti-depressant status of the mother, and a few others).</li>
</ol>
<p>DNA methylation is another epigenetic regulatory mechanism, and <a href="https://10.1101/cshperspect.a023044">the placenta has been recognized for its unique epigenetic landscape</a>. The sncRNA expression data observed will be integrated with sample-matched DNAm data to investigate whether any correlation in expression exists between the two gene regulation mechansisms. <a href="https://10.1098/rsob.180081">The placental gene expression profile also resembles that of cancer</a>, and the top sncRNAs of interest will be examined for their connection to cancer progression.</p>
<p><strong>Software</strong>: R Programming. Some of my code is available on my <a href="https://github.com/nikita-telkar/Projects">GitHub</a>.</p>
<p><strong>Project Highlights</strong>:</p>
<ul>
<li>Self-taught R, RMarkdown, and RNA-seq data analysis
<ul>
<li>quality control, normalization, PCA, linear modeling, DNAm correlation, gene/pathway enrichment</li>
</ul></li>
<li>First instance of large-scale trancriptomic sequencing of the human placenta.<br />
</li>
<li>Establishment of the normative placental transcriptomic profile by several inherent biological variables, and extrinsic technical and environmental variables.<br />
</li>
<li>Correlation to some cardiovasuclar and cancer pathway genes<br />
</li>
<li>Pilot analysis showed that there indeed are microRNAs differentially-expressed by trimester</li>
</ul>
<p><strong>Communication</strong>:</p>
<p><a href="https://drive.google.com/file/d/1OL7ODzJILB5Q56cb9N2DDmLlxZmwdvTz/view?usp=sharing">Project Overview</a><br />
<a href="https://drive.google.com/file/d/1K4JreNCGIXvbKClrT0TwzAF8Sf3B23Yh/view?usp=sharing">BCCHR: Healthy Starts Research Day 2021 - Poster</a></p>
<hr />
</div>
<div id="collaborations" class="section level4">
<h4>Collaborations</h4>
<p><br></p>
<div id="an-ibd-mimic-human-colonoid-chronic-injury-model" class="section level5">
<h5>1. An IBD-mimic human colonoid chronic-injury model</h5>
<p>Intestinal cells undergo regeneration after acute injury. However, the feasibility and capacity of these cells to undergo repeated renewal and regeneration on chronic injury is still not well understood. Using intestinal epithelial cells collected from 2 human patients, colonoid organoids were generated in the lab and were further subjected to rounds of induced damage, inflammation and rescue to observe the renewal mechanisms of these cells</p>
<p><strong>I was recruited as a Bioinformatician for this exploratory project by the <a href="https://www.bcchr.ca/tsteiner">Ted Steiner’s lab at BC Children’s Hospital Research Institute</a> to analyze their RNA-seq data, for which I was given second-authorship on the respective manuscript</strong></p>
<p><strong>Data/Software</strong>: mRNA-seq / R, GSEA</p>
<p><strong>Project Highlights</strong>:</p>
<ul>
<li><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(21)01798-8">Second-author publication in Cell Reports</a></li>
</ul>
<hr />
</div>
<div id="role-of-nf-activated-t-cells-in-the-pathogenesis-of-coxsackievirus-induced-myocarditis" class="section level5">
<h5>2. Role of NF activated T-cells in the Pathogenesis of Coxsackievirus-induced Myocarditis</h5>
<p><em>In progress</em></p>
<p><strong>Data/Software</strong>: murine mRNA-seq / R, GSEA</p>
<hr />
</div>
<div id="telmisartan-treated-tumours-show-an-improved-response-to-radiaition-therapy" class="section level5">
<h5>3. Telmisartan treated tumours show an improved response to radiaition therapy</h5>
<p><em>In progress</em></p>
<p><strong>Data/Software</strong>: human + murine single-cell RNA-seq / R</p>
<hr />
</div>
</div>
</div>
<div id="masters-project" class="section level3">
<h3>Master’s Project</h3>
<p><br></p>
<p><strong>Comparing the Genetic Architecture of Lipid Traits between Populations</strong></p>
<p>The majority of Genome-wide Association Studies, even now, focus on Caucasian, European populations. Genetic differences exist people of differing populations, where people of a certain ancestry or ethnicity can have genetic variants not present in those of European popualtions. These variants can lead to variable gene expression, as well as a differential response to drugs admistered. Hence, the resulsts of these single-poulation centric studies cannot be extrapolated to different populations. There is, therefore, an increased need to conduct such genome-wide studies in several other populations, along with accounting for population differences in large-scale studies.</p>
<p><strong>I collected GWAS summary statistics of three lipid (cholesterol) biomarkers from seven different populations (British, two Greek isolates, Chinese, Japanese, East Asian, Ugandan) calculated the Polygeneic Risk Scores (PRSs) of each individual using the commonly overlapping lipid SNPs to assess if these scores varied across the populations. The heritability as well as trans-ancestral correlation was also calculated. The measured blood lipid biomarker levels were then examined across the genetic scores, to investigate the correlation between the blood lipid levels of one biomarker against the genetic score of another (biomarker-score cross association).</strong></p>
<p><strong>Software</strong>: UNIX, command-line R, PLINK, GEMMA, Popcorn</p>
<p><br></p>
<p><strong>Results</strong>:</p>
<ul>
<li>The majority of European CVD/lipid loci overlap with the Japanese, Chinese, Greek-isolate, and African Ugandan populations.<br />
</li>
<li>HDL biomarker/score showed an inverse relationship with LDL biomarker score; LDL and Triglycerides had a linear relationship, except for in the Ugandan population where triglyceride score did not correlate with biomarker score.<br />
</li>
<li>The two Greek-isolate populations showed near perfect heritability and trans-ethnic correlation with the UK population, same as the Japanese and Chinese population with the East Asian population.</li>
</ul>
<p><strong>Project Highlights</strong>:</p>
<ul>
<li>Opportunity to work as as a Visiting Scientist at the <a href="https://www.sanger.ac.uk/">Wellcome Sanger Institute</a><br />
</li>
<li>Second-author publication in <a href="https://doi.org/10.1038/s41467-019-12026-7">Nature Communications</a><br />
</li>
<li>PRSs available on EMBL-EBI’s <a href="http://www.pgscatalog.org/publication/PGP000046/">PGS Catalog</a><br />
</li>
<li>First comparison study of its kind: Lipid phenotype (lipid blood levels) was found to be associated with lipid genotype (PRS)</li>
</ul>
<hr />
</div>
<div id="undergraduate-summer-research" class="section level3">
<h3>Undergraduate Summer Research</h3>
<p><br></p>
<p><strong>Using WNT5A Expression to Characterize Development in the Human Gut</strong></p>
<p>The <a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2001-3-1-reviews3001">WNT genes</a> are known to participate in genetic pathways involved in cell patterning, signalling, and proliferation during development. WNT5A in particular is invloved in a number of <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=WNT5A">syndromes</a> which are caused by abnormal cell signalling or growth. The gut (made up of the esophagus, intestines, and anus) is one of the organs that grows and elongates the most during embryonic develpement.</p>
<p><strong>Using immunohistochemistry, I measured the presence of the WNT5A protein the human embryonic gut at <a href="https://embryology.med.unsw.edu.au/embryology/index.php/Carnegie_stage_20">Carnegie Stage 20 (~50 days post conception)</a>.</strong></p>
<p><br></p>
<p><strong>Project Highlights</strong>:</p>
<ul>
<li>Received the UK-wide competitive <a href="https://genetics.org.uk/grants/summer-studentships/">Genetics Society Summer Studentship Award</a> 2016.<br />
</li>
<li>Trained in MAPaint and Amira software on Linux, which involved transferring the wet-lab protein expression results into a computer-based 3D model.<br />
</li>
<li>Expression data and results of the project uploaded to the NIH-funded HuDSeN database under the <a href="http://hdbratlas.org/organ-systems/gi-tract/gi-models.html">Human Developmental Biology Resource (HDBR)</a>.<br />
</li>
<li><a href="https://genetics.org.uk/wp-content/uploads/2017/11/15027387_10211122879777688_5581119349267154163_n.jpg">Attended the Genetics Society Summer Workshop</a> to present my research and the report of the same was selected to be published in the <a href="https://genetics.org.uk/wp-content/uploads/2017/09/Issue-76-_-January-2017.pdf">Genetics Society January 2017 Newsletter</a>.<br />
</li>
<li><a href="https://drive.google.com/file/d/1jmUJNHydDgV3-tvmsTmlrqDHbtkCfDcq/view?usp=sharing">Research poster</a> won awards at two separate <a href="https://research.ncl.ac.uk/expeditionresearchscholarships/prizewinners/2016winners/">University competitions</a>.</li>
</ul>
<hr />
<div class="tocify-extend-page" data-unique="tocify-extend-page" style="height: 0;">

</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4,h5,h6",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
